core/model/datasource.py

```diff
@@ -57,7 +57,7 @@ class DataSource(Base, HasSessionCache, DataSourceConstants):
     )
 
     # One DataSource can generate many Editions.
-    editions = relationship("Edition", backref="data_source")
+    editions = relationship("Edition", back_populates="data_source")
```

Comment:
Should this also change to `uselist` too

Reply:
I suppose editions are not iterated over in the rest of the checked code so mypy is not complaining. I can add it if you think it makes sense for future use.

Reply:
I think it does, as long as it doesn't break any tests.